src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -0,0 +1,121 @@
+import styled, { css, keyframes } from "styled-components";
+
+// --- Avatar
+type TAvatar = {
+  backgroundImage?: string;
+};
+export const Avatar = styled.div`
+  background-position: center;
+  background-size: cover;
+  width: 4vh;
+  height: 4vh;
+  background-image: url(${(props: TAvatar) => props?.backgroundImage ?? ""});
+`;
+
+// --- Reaction
+const reactionOffset = "-20px";
+const reactionLeft = css`
+  left: ${reactionOffset};
+`;
+const reactionRight = css`
+  right: ${reactionOffset};
+`;
+
+type TReaction = {
```

Comment:
nit: wondering about the Hungarian notation here; do you recommend propagating this across the codebase? if so is there a lint rule?

Reply:
Its not something im pushing for, rather something im familiar with..

So the prefix _T_ for type and _I_ (upper case i) for interface, followed by the component name

Reply:
Catching back up here, it is inconsistent with the rest of the code, so let's remove it.. would be great if we could migrate everything but won't be possible unfortunately.